eZ/Bundle/EzPublishCoreBundle/EventListener/RequestEventListener.php

```diff
@@ -82,6 +82,9 @@ public function onKernelRequestForward(GetResponseEvent $event)
                     $request->getContent()
                 );
                 $forwardRequest->attributes->add($request->attributes->all());
+                if ($request->headers->has('X-User-Hash')) {
+                    $forwardRequest->headers->set('X-User-Hash', $request->headers->get('X-User-Hash'));
+                }
```

Comment:
the UserContextSubscriber is using the event listener mechanism, but its not a symfony framework event listener - its hooked into a special HttpCache kernel that triggers events during the hash process. i don't think the two can be mixed.

are these things specific to ez publish? or should we provide this listener in FOSHttpCache to make forward/redirect use the correct headers? that would also solve the configuration question...